slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import io.sentry.spring.EnableSentry;
 
 @SpringBootApplication
+@EnableSentry(dsn = "https://b579813c296046399a14368064320494@o938867.ingest.sentry.io/5888744")
```

Comment:
Isn't it possible to get this value from config? As I seems to be unique for each user